prime-router/docs/design/design/HL7v2-FHIR-Inventory.md

```diff
@@ -28,6 +28,26 @@ either store
 HL7 data not translated to FHIR or information on where a particular piece of a FHIR bundle should be mapped to in the
 HL7 message.
 
+### Use of extensions to capture HL7 data
+
+The FHIR bundles that are produced the conversion process are then translated back to HL7 for receivers that want that
+format. In order to ensure:
```

Comment:
The wording of `In order to ensure:` feels awkward but I can't quite figure out what to recommend instead